easyrsa3/easyrsa

```diff
@@ -432,6 +433,13 @@ $out"
 	return 0
 }
 
+verify_curve_ed() {
```

Comment:
This shouldn't just check that the name of the curve is "ed25519" or "ed448", but also that the openssl binary supports eddsa (like the verify_curve_ec() funcion above)

You could use something like (copying form the code above)

```
	if ! "$EASYRSA_OPENSSL" genpkey -algorithm "$EASYRSA_CURVE" > /dev/null; then
		die "Curve $EASYRSA_CURVE not found."
	fi
```